qt/scientific_interfaces/Inelastic/Analysis/DockWidgetArea.cpp

```diff
@@ -35,7 +35,7 @@ IndirectDockWidgetArea::IndirectDockWidgetArea(QWidget *parent) : QMainWindow(pa
   resizeDocks({m_fitPropertyBrowser, plotViewArea}, {20, 20}, Qt::Horizontal);
 }
 
-void IndirectDockWidgetArea::setFitDataView(IndirectFitDataView *fitDataView) {
+void DockWidgetArea::setFitDataView(FitDataView *fitDataView) {
   QDockWidget *dataViewArea = new QDockWidget();
```

Comment:
Apart from the main intention of this PR: shouldn't these QWidgets declared with `new` have their parent assigned? To be exact - `InelasticFitPropertyBrowser, QDockWidget, FitPlotView, QFrame, QDockWidget`